hebsafeharbor/identifier/signals/hebrew_city_recognizer.py

```diff
@@ -13,7 +13,8 @@ class AmbiguousHebrewCityRecognizer(LexiconBasedRecognizer):
     """
 
     DEFAULT_CONFIDENCE_LEVEL = 0.2  # expected confidence level for this recognizer
-    LOCATION_OVERLAP_FACTOR = 0.8  # enhancement factor for this recognizer if overlap with another geo entity found
+    LOCATION_OVERLAP_FACTOR = 0.4  # enhancement factor for this recognizer if overlap with another geo entity found
+    CONTEXT_ENHANCEMENT_FACTOR = 0.4  # enhancement factor for this recognizer if supportive context found
```

Comment:
Consider making this a class parameter with a default value

Reply:
updated